includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -421,4 +419,96 @@ private function getPartnerSlot() {
 		}
 		return null;
 	}
+
+	private function getLogo() {
+		if ( $this->isWikiaOrgCommunity() === true ) {
+			return $this->getLogoWikiaOrg();
+		}
+
+		return [
+			'header' => [
+				'type' => 'link-image',
+				'href' => $this->getHref( 'fandom-logo' ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "fandom-logo" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)